extensions/amp-form/0.1/form-validators.js

```diff
@@ -468,10 +468,16 @@ export function isCheckValiditySupported(doc) {
  * @return {?string}
  */
 function getInvalidType(input) {
+  // 'badInput' takes precedence over others.
+  const validityTypes = ['badInput'];
   for (const invalidType in input.validity) {
-    if (input.validity[invalidType]) {
-      return invalidType;
+    // add other types after
+    if (!validityTypes.includes(invalidType)) {
+      validityTypes.push(invalidType);
     }
   }
-  return null;
+  // Finding error type with value true
+  const response = validityTypes.find(type =>
```

Comment:
@yomeshgupta we don't allow `array.find` due to (lack of) full browser of support and polyfilling it adds unneeded bytes.
You can use `.filter` and take `the first item || null` or do another loop.

Reply:
Hi @aghassemi I have replaced the `.find` with `.filter` and returning the `first element` of the `array` if present else `null`. Let me know if anything else is required. Thanks!